### PR TITLE
add opensearch-dashboard plugin based Dockerfile for adding / updatin…

### PR DESCRIPTION
**Pull request by YANG-DB**
# Changes

add opensearch-dashboard plugin based Dockerfile for adding / updating the dashboards plugin

see (https://opensearch.org/docs/1.2/dashboards/install/plugins/)

## Merge Requirements

For new features contributions please make sure you have completed the following
essential items:

* [ ] `CHANGELOG.md` updated to document new feature additions
* [ ] Appropriate documentation updates in the [docs][]
* [ ] Appropriate Helm chart updates in the [helm-charts][]

Maintainers will not merge until the above have been completed. If you're unsure
which docs need to be changed ping the
[@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers).

[docs]: https://opentelemetry.io/docs/demo/
[helm-charts]: https://github.com/open-telemetry/opentelemetry-helm-charts

### CODE REVIEW

File: src/opensearch-dashboards/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Use OpenSearch image as base
+FROM opensearchproject/opensearch-dashboards:2.7.0
```
Comment (joshuali925) [inline src/opensearch-dashboards/Dockerfile:2]: can this be 2.8.0?

File: src/opensearch-dashboards/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Install plugin (https://opensearch.org/docs/1.2/dashboards/install/plugins/)
+USER root
+RUN sudo /usr/share/opensearch-dashboards/bin/opensearch-dashboards-plugin install <plugin-url>
```
Comment (joshuali925) [inline src/opensearch-dashboards/Dockerfile:6]: would users have to manually build plugin, upload to some server, update `<plugin-url>`? what if users don't need to change plugin from official image?

Also if plugin exists, should uninstall first then install

File: src/opensearch-dashboards/Dockerfile (hunk 1 above)
Comment (YANG-DB) [reply src/opensearch-dashboards/Dockerfile:2]: this should corresponds to the OpenSearch server...

File: src/opensearch-dashboards/Dockerfile (hunk 2 above)
Comment (YANG-DB) [reply src/opensearch-dashboards/Dockerfile:6]: users than need changes to the OpenSearch-dashboards can use this template
https://opensearch.org/docs/1.2/dashboards/install/plugins/#install - same api for create/update plugin

**Pull request closed without merging**